webv2/web.go

```diff
@@ -940,98 +940,95 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 }
 
 func parentTableHelper(table string, update bool) *TableInterleaveStatus {
-	tableInterleaveStatus := &TableInterleaveStatus{Possible: true}
```

Comment:
Interleave tab showing wrong parent table name and the css for Interleaved: at the top in header is also messed up: https://drive.google.com/open?id=15ObNMC8aOSjnK12Rl4uDkattqNzgLZBU&authuser=0&resourcekey=0-He1TRQDQPE2ScXqT2wwntA&usp=drive_link

Reply:
After changing the primary key order and interleaving the parent table is not as expected

Reply:
Updated